common/core/gpudevice.cpp

```diff
@@ -52,6 +54,7 @@ bool GpuDevice::Initialize() {
 
   thread_stage_lock_.unlock();
   HandleHWCSettings();
+  FrameBufferManager::CreateInstance(display_manager_->GetFD());
```

Comment:
You can remove Create Instance now, have a unique ptr for FrameBufferManager here.

Reply:
I am not really familiar with C++ use of smart pointers, but wouldn't it require new FrameBufferManager to be passed in as a public accessible constructor? Currently, std::unique_ptr<FrameBufferManager> frame_buffer_manager_(new FrameBufferManager()) wouldn't work since its privately used through CreateInstance. Is there a known way to circumvent this?